src/org/etools/j1939_84/modules/SummaryModule.java

```diff
@@ -38,16 +38,6 @@ public void addOutcome(int partNumber, int stepNumber, Outcome outcome, String m
         stepResult.addResult(new ActionOutcome(outcome, message));
     }
 
-    public void beginPart(PartResult partResult) {
-        getPartResults().add(partResult);
-    }
-
-    public void endStep(StepResult stepResult) {
-        if (stepResult.getOutcome() == Outcome.INCOMPLETE) {
```

Comment:
I think the `SummaryModule` needs to remain or we need to move this logic to the PartController to add a `PASS` Outcome when the `StepController` has completed without any other outcome.

Reply:
I moved the logic into the PartController to make life simpler.